pkg/scheduler/core/generic_scheduler.go

```diff
@@ -282,7 +282,7 @@ func (g *genericScheduler) processPreemptionWithExtenders(
 ) (map[*v1.Node]*schedulerapi.Victims, error) {
 	if len(nodeToVictims) > 0 {
 		for _, extender := range g.extenders {
-			if extender.SupportsPreemption() {
+			if extender.SupportsPreemption() && extender.IsInterested(pod) {
```

Comment:
In "both **pods** can preempt pod_3, pod_4, pod_5 (priorities in order) " and "if pod_3 and pod_5 on the same **pod**" you mean node, right?

Reply:
In "both **pods** can preempt pod_3, pod_4, pod_5 (priorities in order); if pod_3 and pod_5 on the same **pod**", what do you mean pods or pod here?

Reply:
I think dup or re-preemption in this case is tolerable by system.

We did talk about information sync in extender side, for now, we left it for extender to handle as it will not break existing logic.

Reply:
> we left it for extender to handle as it will not break existing logic.

Extender need to get Pod's latest status to "handle" it (e.g. nominatedNode and deleteTimestamp from apiserver), we can not make sure extender get the latest status when we schedule next pod. That's similar case that we introduced `Bind`. Do we know who's the user of preemption extender? I'm ok to keep current behaviour if no user or the user accept that; as I'd like to keep it simple enough.

Reply:
> In "both pods can preempt pod_3, pod_4, pod_5 (priorities in order); if pod_3 and pod_5 on the same pod", what do you mean pods or pod here?

@xiaoxubeii , **pods** is pod_1 and pod_2; **pod** should be **node**, typo :)

Reply:
cc @ravigadde for user input

Reply:
@resouer My experience has been with one extender only. Diff seems reasonable.